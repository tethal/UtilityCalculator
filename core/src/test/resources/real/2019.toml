[general]
start_date = 2019-05-01
end_date = 2019-12-31
tenant = ["Marie Černá", "Jindřišská 16", "111 50 Praha 1"]
owner = ["Jan Novák", "majitel@example.com"]
report_place = "V Praze"
report_date = 2020-05-01
sources = ["Vyúčtování služeb od Společenství vlastníků za rok 2019"]

[[deposits]]
description="duben - srpen"
count=5
amount=4000

[[deposits]]
description="zari - listopad"
count=3
amount=5000

[[heating]]
annual_cost = 7855.5
start_date = 2019-01-01
end_date = 2019-12-31

[[other_fees]]
start_date = 2019-01-01
end_date = 2019-12-31
annual_cost = 8196

[[cold_water.reading]]
meter_id = "1"
reading_date = 2019-05-01
state = 86

[[cold_water.reading]]
meter_id = "1"
reading_date = 2020-01-01
state = 122


[[cold_water.tariff]]
start_date = 2019-01-01
end_date = 2019-12-31
unit_amount = 82.3025

[[hot_water.reading]]
meter_id = "1"
reading_date = 2019-05-01
state = 50

[[hot_water.reading]]
meter_id = "1"
reading_date = 2020-01-01
state = 71

[[hot_water.tariff]]
start_date = 2019-01-01
end_date = 2019-12-31
unit_amount = 83.512789

[[hot_water.heating_basic_cost]]
start_date = 2019-01-01
end_date = 2019-12-31
annual_cost = 1666.5

[[hot_water.heating_consumable_tariff]]
start_date = 2019-01-01
end_date = 2019-12-31
unit_amount = 135.97404
