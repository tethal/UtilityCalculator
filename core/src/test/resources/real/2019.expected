Report:
  dateRange: 2019-05-01...2020-01-01
  tenant: Marie Černá, Jindřišská 16, 111 50 Praha 1
  owner: Jan Novák, majitel@example.com
  placeDate: V Praze, 2020-05-01
  sources: Vyúčtování služeb od Společenství vlastníků za rok 2019
  sections:
    Studená voda: 2962.89
    Teplá voda: 5720.22
    Vytápění: 3299.31
    Ostatní poplatky: 5464.00
    Přijaté zálohy: -35000
    TOTAL: -17553.58

Studená voda:
  readings:
    [1] 2019-05-01...2020-01-01: 86...122 = 36
  priceList:
    2019-05-01...2020-01-01: 36.000 * 82.3025 = 2962.89

Teplá voda:
  readings:
    [1] 2019-05-01...2020-01-01: 50...71 = 21
  priceList:
    2019-05-01...2020-01-01: 21.000 * 83.512789 = 1753.77
  heatingBasicParts:
    2019-05-01...2020-01-01: 8.00 * 138.88 = 1111.00
  heatingConsumableParts:
    2019-05-01...2020-01-01: 21.000 * 135.97404 = 2855.45

Vytápění:
  2019-05: 1.00 * 7855.5 * 0.02 = 157.11
  2019-06: 1.00 * 7855.5 * 0.0 = 0.00
  2019-07: 1.00 * 7855.5 * 0.0 = 0.00
  2019-08: 1.00 * 7855.5 * 0.0 = 0.00
  2019-09: 1.00 * 7855.5 * 0.01 = 78.56
  2019-10: 1.00 * 7855.5 * 0.08 = 628.44
  2019-11: 1.00 * 7855.5 * 0.14 = 1099.77
  2019-12: 1.00 * 7855.5 * 0.17 = 1335.44

Ostatní poplatky:
  2019-05-01...2020-01-01: 8.00 * 683.00 = 5464.00

Přijaté zálohy:
  duben - srpen: 5 * 4000 = 20000
  zari - listopad: 3 * 5000 = 15000
